---
title: Macro Methods
layout: default
group: courses
---

# Biophysics 204B: Methods in Macromolecular Structure #


## Winter 2026 Syllabus ##

<img class="img-fluid mx-auto d-block" src="/static/img/aeronab.jpg" style="height: 300px" alt="What is the next experiment?">

**Course Title:** Methods in Macromolecular Structure

**Course Format:** 4 hours of lecture/group work per week in class, substantial group work outside of class hours

**Location and Date/Hours:** Tuesday, 9:00-10:30AM , Thursday 9:00-11:30AM in Genentech Hall 227

**Prerequisites:** All incoming first-year BP graduate students are required to enroll in this course.

**Grading:** Letter grade

**Textbook:** None. Lab protocols and course materials will be available in class or online

**Instructors:** [Willow Coyote-Maestas](mailto:willow.coyote-maestas@ucsf.edu),[John Gross](mailto:jdgross@cgl.ucsf.edu), [Tom Goddard](mailto:goddard@sonic.net), [Dan Southworth](mailto:Dan.Southworth@ucsf.edu)

**TAs:**
- [Keana Davis](mailto:Keana.Davis@ucsf.edu)
- [Justin Sim](mailto:Sustin.Sim@ucsf.edu)

**Lecturers/Facilitators:**

Willow Coyote-Maestas, John Gross, Tom Goddard, Dan Southworth


**Background:**

Fluency in multiple biophysical methods is often critical for answering mechanistic questions. Traditionally, students are exposed to the fundamentals of multiple techniques through lectures that cover the theory before exposure, and, for some, in analysis or data collection during lab rotations. However, this structure means that only students who rotate in specific labs gain hands-on exposure, which could limit adventurous experiments in future years. To train the next generation of biophysicists at UCSF, we have decided to alter this traditional structure by creating  “Macromolecular Methods”, a class that places emphasis on playing with data. We have designed Macromolecular Methods to be a team-based class where students develop their own analysis of real data that, in non-pandemic years, they have collected. The goal of this course is to provide students with sufficient exposure to major methods in biophysics to enable them to pursue these more fully in their independent research.

**Course Description:**

This is a team-based class where students work in small groups to develop their own analysis of real data. Statistical aspects of rigor and reproducibility in structural biology will be emphasized throughout lectures, journal club presentations, and hands-on activities. The website for the [2017](/courses/methods_2017/),  [2018](/courses/methods_2018/), [2019](/courses/methods_2019), [2020](/courses/methods_2020) editions are available online.

*Course expectations*: This course will likely be different from most other courses you have taken before. Our expectation is that you will maximize your learning by being an active participant - there is significant out-of-class work required to enable this. Many learning opportunities will arise if you are willing to take advantage of the hands-on experience and from those who take the time to show you interesting aspects of the various methods. Data analysis will not follow a linear path - our expectation is that you start to become comfortable with unknowns and messy results. It is likely that you will encounter issues using the various scientific software required for the course - while we will help troubleshoot, navigating these challenges effectively is a core expectation that we want to see students build towards during the course.  ***Please let us know via slack if you will be unable to make any classes.*** 

*Ethics*: This course is more than a training experience; data analysis is part of ongoing active research projects, the results of which will be published to the broader scientific community. The community must be able to understand our work, replicate it, and have confidence in its findings. We must therefore ensure the integrity of the information we disseminate. To do so, it is essential that students perform and document their experiments and analyses as faithfully as possible. Mistakes and oversights are normal and to be expected, but they must not be ignored, concealed, or disguised. 

*Respect*: This course is built around an open research project performed in teams. Successful completion of the course objectives will require that students work together effectively, so please respect the time and effort of your classmates and instructors. Moreover, as part of the research process, we will consider and debate a variety of ideas and approaches; however, we must not allow our position on a particular idea or argument to compromise our respect for its author. We therefore expect course participants to give all instructors and students, regardless of academic or personal background, their complete professional respect; anything less will not be tolerated.

*Accommodations for students with disabilities*: The Graduate Division embraces all students, including students with documented disabilities. UCSF is committed to providing all students equal access to all of its programs, services, and activities. Student Disability Services (SDS) is the campus office that works with students who have disabilities to determine and coordinate reasonable accommodations. Students who have, or think they may have, a disability are invited to contact SDS (StudentDisability@ucsf.edu); or 415-476-6595) for a confidential discussion and to review the process for requesting accommodations in classroom and clinical settings. More information is available online at http://sds.ucsf.edu. Accommodations are never retroactive; therefore, students are encouraged to register with Student Disability Services (http://sds.ucsf.edu/) as soon as they begin their programs. UCSF encourages students to engage in support seeking behavior via all of the resources available through Student Life, for consistent support and access to their programs.

The course instructors and teaching assistants value the contributions, ideas and perspectives of all students. It is our intent that students from different backgrounds be well-served by this course, that students' learning needs be addressed both in and out of class, and that the diversity that the students bring to this class be viewed as a resource, strength and benefit. Although the instructors are committed to continuous improvement of our practices and our learning environment, we value input from students and your suggestions are encouraged and appreciated. Please let the course director or program leadership know ways to improve the effectiveness of the course for you personally, or for other students or student groups. 

### 2026 schedule

<iframe src="https://calendar.google.com/calendar/embed?src=0d44412d8f7979e56b443a878f7d9282f177e38062f5b332f1fad0feff111b1c%40group.calendar.google.com&ctz=America%2FLos_Angeles" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>

### TEAM ASSIGNMENTS

**Team name: 1**
   - Jess Flowers
   - Delia Gagnon
   - Devin Hemmans

**Team name: 2**
   - Jocelyn Rios
   - Beau Lonnquist
   - Keerthi Renduchintala

**Team name: 3**
   - Beatrice Mihalache
   - Malavika Vinod
   - Taylor Womack

<a name="Jan6"></a>
### Tuesday, January 6 
#### 9:00-10:30AM Introduction to the Course and Software Check

- Software check, ChimeraX, NMRBox
- Installation and registration:
- [ChimeraX daily build](https://www.rbvi.ucsf.edu/chimerax/download.html#daily-release)
- [Register for NMRHub](https://nmrhub.org)
- [Join UCSFMacroMethods Course Hosted by NMRHub](https://urldefense.com/v3/__https://nmrbox.nmrhub.org/events/events/2026-ucsfmacromethods/eyJzbHVnIjoiMjAyNi11Y3NmbWFjcm9tZXRob2RzIiwiZnVuY3Rpb24iOiJwcml2YXRlLWV2ZW50In0.VmMVFGVV9-5w0FXXq0XDRzjBY8M__;!!LQC6Cpwp!ut7U1gweMZKBo9wKrD92p7sVlQEuGZzQYm5CAyrvGoxXE9fAYaKp_ri_cXGNhE1KzouU1C2s8ak4jCb3AI9owTc$)
- [Download vncViewer](https://www.realvnc.com/en/connect/download/viewer/?lai_vid=MM0wp9QQlt4jK&lai_sr=5-9&lai_sl=l&lai_p=1)
- [CCP4/Phenix/ADXV/EMan2 download tutorial](https://docs.google.com/document/d/1pRA-pERFvX05yrGZcZjfugxiV5YDyrFsX64yHhcyOmw/edit#)
     	
<a name="Jan8"></a>
## Jan 8-20 Structure prediction and design with Boltz 2 and BoltzGen and ChimeraX (Tom Goddard)

Will look at how to predict molecular structures (proteins, nucleic acids, ligands) using Boltz 2 and how to design proteins using BoltzGen, and how to visualize and assess confidence scores of prediction results with ChimeraX. 

Most of the time will be spent working on team projects. There will be 3 teams of 3 students each. Here are 3 project ideas:

- Group 1: [ABC transporter drug interaction predictions](https://www.rbvi.ucsf.edu/chimerax/data/macromethods-jan2026/macromethods.html#abc)
- Group 2: [Transcription factor protein-nucleic acid interactions](https://www.rbvi.ucsf.edu/chimerax/data/macromethods-jan2026/macromethods.html#tf)
- Group 3: [Designing a therapeutic protein/peptide for fibril aggregate diseases](https://www.rbvi.ucsf.edu/chimerax/data/macromethods-jan2026/macromethods.html#pe)

See Tom's website link [here](https://www.rbvi.ucsf.edu/chimerax/data/macromethods-jan2026/macromethods.html).

### Thursday January 8 
#### 9:00-11:30AM Making and evaluating Boltz predictions in ChimeraX on laptop computer, learning how to look at confidence scores, possibly learning how to use Boltz server for larger predictions

- 9:00-9:15, Journal Club: Group 1, [Boltz-2: Towards Accurate and Efficient Binding Affinity Prediction](https://www.biorxiv.org/content/10.1101/2025.06.14.659707v1)
Saro Passaro, Gabriele Corso, Jeremy Wohlwend, Mateo Reveiz, Stephan Thaler, Vignesh Ram Somnath, Noah Getz, Tally Portnoi, Julien Roy, Hannes Stark, David Kwabi-Addo, Dominique Beaini, Tommi Jaakkola, Regina Barzilay
June 14, 2025 bioRxiv preprint
- 9:15-11:30 Making and evaluating Boltz predictions in ChimeraX on laptop computer, learning how to look at confidence scores, possibly learning how to use Boltz server for larger predictions. Please have ChimeraX daily build installed (Jan 2026) on your laptops. See in depth on Tom's website here: [Details](https://www.rbvi.ucsf.edu/chimerax/data/macromethods-jan2026/day1.html)

- Some past tutorials:
- [Winter 2025 class: Exploring atomic models, NMR and cryoEM data, and Foldseek results with Chimera](https://www.rbvi.ucsf.edu/chimerax/data/macromethods-jan2025/chimerax_intro.html)
- [Color by Chemical Shift Perturbations](https://www.rbvi.ucsf.edu/chimerax/data/nanobody-feb2021/chemshift.html)
- [How to View Nanobody Molecular Dynamics in ChimeraX](https://www.rbvi.ucsf.edu/chimerax/data/nanobody-feb2021/nanobody_md.html)
- [ChimeraX tutorial by Tom Goddard](https://www.rbvi.ucsf.edu/chimerax/data/macromethods-jan2025/chimerax_intro.html)
- [More generic instructions for drug screening with AF3](https://www.rbvi.ucsf.edu/chimerax/data/af3-drugs-dec2024/af3_drugs.html)
  
<a name="Jan13"></a>
### Tuesday January 13
#### 9:00-10:30AM Introduction to 3 team projects (Tom Goddard)

- 9:00-9:15AM, Journal Club: Group 3, [BoltzGen: Toward Universal Binder Design](https://www.biorxiv.org/content/10.1101/2025.11.20.689494v1)
Hannes Stark, Felix Faltings, MinGyu Choi, Yuxin Xie, Eunsu Hur, Timothy O'Donnell, Anton Bushuiev, Talip Uçar, Saro Passaro, Weian Mao, Mateo Reveiz, Roman Bushuiev, Tomas Pluskal, Josef Sivic, Karsten Kreis, Arash Vahdat, Shamayeeta Ray, Jonathan T. Goldstein, Andrew Savinov, Jacob A. Hambalek, Anshika Gupta, Diego A. Taquiri-Diaz, Yaotian Zhang, A. Katherine Hatstat, Angelika Arada, Nam Hyeong Kim, Ethel Tackie-Yarboi, Dylan Boselli, Lee Schnaider, Chang C. Liu, Gene-Wei Li, Denes Hnisz, David M. Sabatini, William F. DeGrado, Jeremy Wohlwend, Gabriele Corso, Regina Barzilay, Tommi Jaakkola. Nov 24, 2025 bioRxiv preprint
- 9:15-10:30, Introduction to 3 team projects
- Team 1: [ABC transporter drug interaction predictions](https://www.rbvi.ucsf.edu/chimerax/data/macromethods-jan2026/macromethods.html#abc)
- Team 2: [Transcription factor protein-nucleic acid interactions](https://www.rbvi.ucsf.edu/chimerax/data/macromethods-jan2026/macromethods.html#tf)
- Team 3: [Designing a therapeutic protein/peptide for fibril aggregate diseases](https://www.rbvi.ucsf.edu/chimerax/data/macromethods-jan2026/macromethods.html#pe)

  
<a name="Jan15"></a>
### Thursday January 15
#### 9:00-11:30AM Group Work

- 9:00-9:15AM, Journal Club: Group 2, [Predicting protein-protein interactions in the human proteome](https://www.science.org/doi/10.1126/science.adt1630)
Zhang J, Humphreys IR, Pei J, Kim J, Choi C, Yuan R, Durham J, Liu S, Choi HJ, Baek M, Baker D, Cong Q. Predicting protein-protein interactions in the human proteome. Science. 2025 Oct
- 9:15-1130AM In class work on team projects 

<a name="Jan20"></a>
### Tuesday January 20,
#### 9:00-10:30AM Group Presentations, 20 minutes + 10 min Q&A, 90 minutes total

- 9:00-9:30AM Team 1
- 9:30-10:00AM Team 2
- 10:00-10:30AM Team 3

<a name="Jan22"></a>
## Jan 22-Jan 27 Deep Mutational Scanning (Willow Coyote-Maestas)
### Thursday, January 22
#### Why do a mutational scan?
- 9:00-9:20AM The library- how it's made and what sorts of variants are studied.
- 9:20-10:40 The host-What organism are you doing the experiments in?
- 9:40-10:00 The assay(s)-What is measured and how do they address questions?
- 10:00-10:10 Break
- 10:10-11:45 Data processing and statistics-How do you find the impact of variants?
- 10:45-11:30 Playing in big data-What is actually happening in the experiment?
- Read before class [Origin of DMS Review](https://www.nature.com/articles/nmeth.3027)
  
<a name="Jan27"></a>
### Tuesday, Jan 27
#### DMS-applications and future opportunities
-  Read before class:
-  [Structural biology](https://www.cell.com/cell/abstract/S0092-8674(24)01373-4)
-  [Genetic variation](https://pmc.ncbi/nlm.nih.gov/articles/PMC6181777)
-  [Evolution](https://www.science.org/doi/10.1126/science.abn6895)
-  [Pharmacology](https://www.nature.com/articles/s41586-024-08533-3)
  
<a name="Jan29"></a>
## Jan 29- Feb 19 Electron Microscopy (Dan Southworth), Practicals Max Tucker, Alex Long, Greg Merz, and Eric Tse
Cryo-EM sample preparation, data collection and reconstruction analysis of the p97 AAA+ complex and its adapter proteins.

Teams will be supported by co-instructors
- Team 1: Alex Long
- Team 2: Max Tucker
- Team 3: Greg Merz

### Thursday, January 29
#### Cryo-EM Sample Preparation (note special classrooms)

- 9:00-9:40AM, Lecture: Introduction to Cryo-EM and Introduction Vitrification
  (Location: GH 227, teaching lab)

- 9:45-11:30AM, Practical: Sample Vitrification & Clipping (Location: Team 1 - Weill Cryo-EM Lab; Team 2 - GH 1st Floor Cryo-EM lab; Team 3 - Sandler 3rd Floor Lab)

- Homework: [Grant Jensen Lectures on Sample Preparation Part 1 and Part4](https://www.youtube.com/playlist?list=PLhiuGaXlZZenm7lu5qv_A59zEWkRKkBn5)

<a name="Feb3"></a>
### Tuesday, February 3
#### Cryo-EM Data Collection (No lecture)

- 9:00-10:30AM, Practical: Cryo-EM Grid screening (Location: Team 1 & Team 2 - Weill Krios Room, Team 2 - Sander 1st Floor)

- Homework: Particle Picking and 2D Processing; [Grant Jensen Lectures Part 2,3 and 4 (3D Reconstruction)](https://www.youtube.com/playlist?list=PLhiuGaXlZZenm7lu5qv_A59zEWkRKkBn5)

<a name="Feb5"></a>
### Thursday, February 5
#### Particle picking/2D Classification

- 9:00-9:25AM, Lecture: Image Formation, 2D Analysis, Review Data Collection, CryoSparc Tutorial (Location: GH 227, teaching lab)

- 9:30-11:30AM, Practical: Micrograph Curation, CTF, Particle Picking

- Homework: 2D Data Processing; [Grant Jensen Lectures Part 6](https://www.youtube.com/playlist?list=PLhiuGaXlZZenm7lu5qv_A59zEWkRKkBn5)

<a name="Feb10"></a>
### Tuesday, February 10
#### Analyses of 3D classes and refinement

- 9:00-9:25AM, Lecture: 3D Reconstruction Basics (Location: GH 227, teaching lab)

- 9:30-10:30AM, Practical: 2D Classification Analysis; Set up 3D Classification

- Homework: Initial Model, 3D Classification 


<a name="Feb12"></a>
### Thursday, February 12
#### Fitting and 3D refinement

- 9:00-9:25AM, Lecture: p97 Structure (GH 227, teaching lab) 

- 9:30-11:30AM, Practical: Combine 3D Classes and Start Refinement
  
- Homework: Continue 3D Refinement, Dock Models

<a name="Feb17"></a>
### Tuesday, February 17
#### Model Building

- 9:00-9:25AM, Lecture: Isolde; symmetry expansion, focus classification
- 9:30-10:30AM Practical: Modeling p97, Look for extra desnity
- Homework: Finalize Map/Model, Identify Adapter

<a name="Feb19"></a>
### Thursday, Feb 19 Final presentations 20 minutes + 10 Q&A

- Group presentations on work, ~20min each: challenges/successes; what is your adapter?

- 9:00-9:30  Team 1
- 9:30-10:00  Team 2
- 10:00-10:30 Team 3

## Supplemental material and tutorial videos

- [Getting started in CryoEM - Grant Jensen lectures](https://www.youtube.com/playlist?list=PL8_xPU5epJdctoHdQjpfHmd_z9WvGxK8-)
- [LMB EM Course](https://www2.mrc-lmb.cam.ac.uk/research/scientific-training/electron-microscopy/)
- [Protein Dynamics by NMR- Dorothee Kern](https://www.youtube.com/watch?v=fzM5V0bbybM)
- [Ligand binding and drug design-Dorothee Kern](https://www.youtube.com/watch?v=7MuBsYNrypg&t=659s)
- [NMR Theory Course , James Keeler](https://www.youtube.com/watch?v=nM7jQFhrvR0&index=1&list=PLE20foNk9J6L1dh9X27RaPiaul8_7wrAY)



<a name="Feb24"></a>
## Feb 24-March 17  Protein NMR , Nanobodies for SARS-CoV-2 Spike- John Gross, Dominic Grisingher and Catherine Kuhn
### JOURNAL CLUB ASSIGNMENTS
- Team 3: 2/26 Schoof (2020) [An ultrapotent synthetic nanobody neutralizes SARS-CoV-2](https://pubs.acs.org/doi/10.1021/ja000350l)
- Team 1: 3/5 Hudjuk (2000) [NMR-Based Screening of Proteins Containing 13C-labeled Methyl Groups](https://www.nature.com/articles/nsb1296-995.pdf)
- Team 2: 3/10 Farmer (1996) [Localizing the NADP+ binding site on the MurB enzyme by NMR](https://www.nature.com/articles/nsb1296-995.pdf)


### Tuesday, Feb 24th, The Fourier Transform and NMR Data processing

- 9:00-10:00AM, [Theory](https://ucsf.box.com/s/e4fv4psxosyzk7k4jp8ezl694c9r7gzk) (JG)
- 10-11:30, [HSQC data processing with NMRPipe, tutorial](https://ucsf.box.com/s/j661e28044xbagcvywoqya9lukh9g19c) (JG)
- [Before class, review Multidimensional NMR](https://ucsf.box.com/s/dcm7msktauh0yyinvc0ocrdfw1i3i9vh)
- Supplemental reading: [How does an HSQC work?](http://cdn.fraserlab.com/courses/methods/HSQC-Guided-tour.pdf)
- [FTexamples](https://ucsf.box.com/s/o7f8qe6n0ytzpde4ap5wtx0ek5ckbkji)
- Watch: [A visual introduction to the Fourier Transform](https://www.youtube.com/watch?v=spUNpyF58BY)
  

<a name="Feb26"></a>
### Thursday, Feb 26th, Process 15N HSQC of nanobody and nanobody Spike complexes
- 9:00-9:15AM, [Journal Club Team 3](https://www.science.org/doi/pdf/10.1126/science.abe3255)
- 9:00-10:30, Work in Groups: [HSQC data processing with NMRPipe, tutorial](https://ucsf.box.com/s/j661e28044xbagcvywoqya9lukh9g19c)

  
<a name="Mar3"></a>
### Tuesday, March 3rd Hands-on Data collection (Dominic Grisingher and Catherine Kuhn)
#### 9:00-11:30AM Acquiring and processing 15N HSQC Data
#### Note special location: groups will meet in NMRLAB GH S106 for data acquisition or Mission Hall 2105 for processing
- 9:00-10:00am Team 1, NMR Lab, Teams 2 and 3 finish processing 15N HSQCs in MH 2105 
- 10:00-11:00 Team 2, NMR Lab, Teams 1 and 3 finish processing 15N HSQCs in MH 2105
- 11:00-noon 	Team 3, NMR Lab, Teams 1 and 2 finish processing 15N HSQCs in MH 2015

- [Acquiring_2D_15N-HSQC_in_TopSpin](https://ucsf.box.com/s/tc0dg14opvz3bfdht92r6pn56h3i7fg4)
- [Processing_Data_in_TopSpin](https://ucsf.box.com/s/6jznx75m14b24xf84ot8cgporudfkf8i)
- [Topspin_Commands](https://ucsf.box.com/s/7gss7koit3a9v0iccm0arukojzkf8u9l)


- Homework: finish processing all NMR data before class on Tuesday 

<a name="Mar5"></a>
### Thursday March 5
#### Introduction to ILV labeling of proteins for methyl group NMR

- Homework due: finish processing all NMR data before class today

- 9:00-9:15AM, [Journal Club Team 1](https://pubs.acs.org/doi/10.1021/ja000350l)
- 9:15-10:30 Process 13C HSQCs of ILV labeled Nanobodies alone and bound to spike RBD
- Read before class: [Ideas for Final Presentations, Aashish's miniquals](https://ucsf.box.com/s/hepo6urx0xlggtdhddo40ks5kcwhzhb5)

<a name="Mar10"></a>
### Tuesday, March 10
#### Chemical shift perturbation (CSP) mapping 

- 9:00-9:15AM, [Journal Club Team 2](https://www.nature.com/articles/nsb1296-995.pdf)
- 9:15-10:00, tutorial on overlay spectra for chemical shift perturbation analysis using Sparky. 
- 10-11:30, overlay HSQC spectra, cross-peak bookkeeping analysis, mapping CSPs onto structures
- [Sparky tutorial](https://ucsf.box.com/s/adtpj0rcrl6ecttsqnsa5fs165qatub5)
- [List of common Sparky commands](https://ucsf.box.com/s/ftp2is8jgyd2f3wizpi0m1vn5rqbfzn7)

- Might be useful: [Color by Chemical Shift Perturbations](https://www.rbvi.ucsf.edu/chimerax/data/nanobody-feb2021/chemshift.html)

<a name="Mar12"></a>
### Thursday, Mar 12th, class meets in GH-S261
- 9:00-9:30AM Question and Answer Catch-up
- 9:30-10:30 wrap-up 13C HSQC processing and CSP analysis

- Might be useful: [Color by Chemical Shift Perturbations](https://www.rbvi.ucsf.edu/chimerax/data/nanobody-feb2021/chemshift.html)

### Materials for TA Office Hours

- [1-an example  13C HSQC of Nb6 , 2-Nb6 and mNb sequences, 3-Sparky tutorial](https://ucsf.box.com/s/boxmdnws7xbg04p812agth005b7augu5)

<a name="Mar17"></a>
### Tuesday, March 17
#### Final Presentations NMR, 20 minutes + 10 min Q&A
- 9:00-9:30 Final Presentation Team 1, How are the dynamics of Nb6 altered by affinity maturation?
- 9:30-10:00 Final Presentation Team 2, Does NMR identify the binding site of SARS-CoV2 Spike for Nb6 and mNb6?
- 10:00-10:30 Final Presentation Team 3, How are the dynamics of Nb6 and mNb6 modulated by binding to Spike?

### Reading on rigor and reproducibility in NMR:

- [Tools for validating NMR structures](http://cdn.fraserlab.com/courses/methods/2014_vuister.pdf)
- [Q-scores](http://cdn.fraserlab.com/courses/methods/2004_clore.pdf)
- [Integrative modeling](http://cdn.fraserlab.com/courses/methods/2019_rout.pdf)
